feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/SearchActivity.kt

```diff
@@ -84,10 +88,15 @@ class SearchActivity : ComponentActivity() {
 
               Spacer(modifier = Modifier.padding(8.dp))
 
-              CommonClaims(
-                selectClaim = viewModel::onSelectClaim,
-                commonClaims = viewState.commonClaims,
-              )
+              val errorMessage = viewState.errorMessage
+              if (errorMessage != null) {
+                Error(errorMessage) { viewModel.loadSearchableClaims() }
```

Comment:
The error message as far as I understand is more often than not something that the user would like to see. Not sure if this case in particular the error is localized properly and is relevant for a normal member to see by the backend, but if not, maybe we can do with a generic network error message as we often do instead.